force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -354,15 +568,26 @@ public with sharing class FlowControlLoopsCollections {
         // add Jane Smith => 60000
         // add Sam Brown => 55000
         // add Alice Johnson => 65000
+        employeeSalaries.put('John Doe', 50000);
+        employeeSalaries.put('Jane Smith', 60000);
+        employeeSalaries.put('Sam Brown', 55000);
+        employeeSalaries.put('Alice Johnson', 65000);
```

Comment:
Since question 20 and 19 uses the same map, try creating  a static map and calling it once, helps to reuse code and not duplicating